src/vs/workbench/parts/files/electron-browser/fileActions.ts

```diff
@@ -1392,6 +1385,33 @@ export function validateFileName(parent: ExplorerItem, name: string, allowOverwr
 	return null;
 }
 
+
+interface IMappedPath {
```

Comment:
Having an interface just for the return type which is used in one place feels like overkill. I would just have this inline.